lib/index.js

```diff
@@ -0,0 +1,68 @@
+import { buildProcessPipeline } from "./util.js";
+import { readFile } from "node:fs/promises";
+import * as path from "node:path";
+
+export const key = "static";
```

Comment:
@FND Do you remember what this name is actually used for? I'm not sure it is used for plugins that are in the default list, only for the ones that are not:
https://github.com/faucet-pipeline/faucet-pipeline-core/blob/main/lib/plugins.js#L5